test/unit/org/apache/cassandra/repair/FuzzTestBase.java

```diff
@@ -799,6 +814,8 @@ private <REQ, RSP> void maybeEnqueue(Message<REQ> message, InetAddressAndPort to
                 CallbackContext cb;
                 if (callback != null)
                 {
+                    if (callbacks.containsKey(message.id()))
```

Comment:
this is to better match `MessagingService`, this doesn't happen but better to be consistent